From: Brad Hubbard <bhubbard@redhat.com>
Date: Tue, 10 May 2016 16:44:44 +1000
Subject: log: Log.cc: Assign LOG_INFO priority to syslog calls

LOG_DEBUG prio messages are not logged by a default syslog
configuration so log at LOG_INFO instead.

Fixes: http://tracker.ceph.com/issues/15808
Signed-off-by: Brad Hubbard <bhubbard@redhat.com>
(cherry picked from commit 3ab5a660a45506d6c4c41bfeb5babdf188e62b3d)

Resolves: rhbz#1378675
---
 src/log/Log.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/log/Log.cc b/src/log/Log.cc
index 29e0529..1d74b00 100644
--- a/src/log/Log.cc
+++ b/src/log/Log.cc
@@ -329,7 +329,7 @@ void Log::_flush(EntryQueue *t, EntryQueue *requeue, bool crash)
       }
 
       if (do_syslog) {
-        syslog(LOG_USER|LOG_DEBUG, "%s", buf);
+        syslog(LOG_USER|LOG_INFO, "%s", buf);
       }
 
       if (do_stderr) {
@@ -367,7 +367,7 @@ void Log::_log_message(const char *s, bool crash)
       cerr << "problem writing to " << m_log_file << ": " << cpp_strerror(r) << std::endl;
   }
   if ((crash ? m_syslog_crash : m_syslog_log) >= 0) {
-    syslog(LOG_USER|LOG_DEBUG, "%s", s);
+    syslog(LOG_USER|LOG_INFO, "%s", s);
   }
 
   if ((crash ? m_stderr_crash : m_stderr_log) >= 0) {
